### ISSUE TITLE
Example for Asp.Net Core with own controllers

### ISSUE THREAD

**Issue by keycad** [jlucansky/Quartzmin#9]
Hi,

I'm trying to use your tool in a project that also hosts some controllers, but I'm having no luck. I'm probably doing something wrong, but if you could give me a hint I'll be grateful.

I have an Asp.Net Core 2.1 web api project, that exposes a couple of controllers. This project also starts the scheduler, and should include your management tool. The problem is, when I add it, I just can see the initial page but nothing from your Content folder (styles, scripts...) is loaded. And besides that, navigation doesn't work either.

Thanks in advance

**Edit:** I'm looking for a similar way to deploy it than CrystalQuartz, that just adding `app.UseCrystalQuartz(() => scheduler)`, it deploys the web in `/quartz` path

**Comment by BaronRector** [comment]
I'm also trying to accomplish the same thing. Did you have any luck?

**Comment by keycad** [comment]
nope... sorry. I was comparing Quartz.NET and Hangfire and not being able to solve this made my choice easier...

**Comment by BaronRector** [comment]
Well that's unfortunate. Thanks for the quick reply! Is there a reason you didn't go with Crystal Quartz?

**Comment by natnat85** [comment]
I have the same issue regarding my application.  it's a web app on a stable IIS that uses MVC, I need quartzmin to run at /quartzmin so that the real application is not override. Was a solution found or is Crystal Quartz the only option here?  QuartzMin does have the better UI and feature set.

**Comment by keycad** [comment]
> Well that's unfortunate. Thanks for the quick reply! Is there a reason you didn't go with Crystal Quartz?

Sorry for the so late reply. The reason is Hangfire's dashboard is much better than Crystal Quartz's (at least it was at that time, when I tested them)

**Comment by edwinn1990** [comment]
Did you manage to find a solution for this?

**Comment by DenisPav** [comment]
I've managed to connect it to my project. I've used forked version that is available on nuget and targets netcoreapp3.1. After that I actually added custom convention and mapped quartzmin controllers to separate route.

It's a bit of work, but after that it works flawlessly.

**Comment by xico002** [comment]
Hi! I´m using .net core 2.2. And I can't see the ui. With CristalQUartz is as simple as app.UseCrystalQuartz(() => scheduler. With quartzmin I do as said in the readme but doesn't work. And also what is the url to access quartzmin after deployment?
Can anyone help me with this?

**Comment by pigiax** [comment]
> I've managed to connect it to my project. I've used forked version that is available on nuget and targets netcoreapp3.1. After that I actually added custom convention and mapped quartzmin controllers to separate route.
> 
> It's a bit of work, but after that it works flawlessly.

@DenisPav did you have "custom convention and mapped quartzmin controllers to separate route." sample code?

**Comment by DenisPav** [comment]
@pigiax Nope, I only have it in a private project. I can probably create an example repo when I catch some time.

**Comment by DenisPav** [comment]
@pigiax 👋 

You can find sample [here](https://github.com/DenisPav/QuartzminSample). Everything is mostly explained there.

**Comment by pigiax** [comment]
@DenisPav thank you so much! Everything it's clear!
